portal-web/test/functional/com/liferay/portalweb/macros/FormPortletConfiguration.macro

```diff
@@ -2,6 +2,15 @@ definition {
 
 	var formName = "Created Form Name";
 
+	macro changeLanguageTo {
+		Click(
+			changeLanguageTo = "${getLanguage}",
```

Comment:
@wesleyroberts, please rename:

- `changeLanguageTo` to `key_language`
- `getLanguage` to `currentLanguage`

Reply:
changes made